worlds/osrs/__init__.py

```diff
@@ -164,11 +164,8 @@ def create_regions(self) -> None:
                 entrance.connect(self.region_name_to_data[parsed_outbound])
 
                 item_name = self.region_rows_by_name[parsed_outbound].itemReq
-                if "*" not in outbound_region_name and "*" not in item_name:
-                    entrance.access_rule = lambda state, item_name=item_name: state.has(item_name, self.player)
-                    continue
-
-                self.generate_special_rules_for(entrance, region_row, outbound_region_name)
+                entrance.access_rule = lambda state, item_name=item_name.replace("*",""): state.has(item_name.replace("*",""), self.player)
```

Comment:
Does the replace have to be in both parts of this? (Genuine question)
```suggestion
                entrance.access_rule = lambda state, item_name=item_name.replace("*",""): state.has(item_name, self.player)
```

Reply:
Probably not. I'm guessing I had it inside the `has` call, then later added the parameter to it as a form of circumventing Lambda Capture and forgot to remove it from the inner one.